releaseNote.md

```diff
@@ -32,7 +32,7 @@ mkdir \actions-runner ; cd \actions-runner
 Invoke-WebRequest -Uri https://github.com/actions/runner/releases/download/v<RUNNER_VERSION>/actions-runner-win-x64-<RUNNER_VERSION>.zip -OutFile actions-runner-win-x64-<RUNNER_VERSION>.zip
```

Comment:
do you mind fixing the `<DRIVE>:\`? it looks weird after it gets rendered.
`We recommend configuring the runner under ":\actions-runner"`

Reply:
Done